changelog/_unreleased/2025-07-31-fix-script-loader-loading-invalid-cache-paths.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Fix ScriptLoader loading invalid cache paths
+author: Benjamin Wittwer
+author_email: Discord.Benjamin@web.de
+author_github: gecolay
+---
+# Core
+* Changed `Shopware\Core\Framework\Script\Execution\ScriptLoader` to no longer load invalid cache paths from the cache
```

Comment:
we probably should mention the cases when this happens, and that not all are affected